core-bundle/src/Controller/ContentElement/DescriptionListController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Controller\ContentElement;
+
+use Contao\ContentModel;
+use Contao\CoreBundle\DependencyInjection\Attribute\AsContentElement;
+use Contao\CoreBundle\Twig\FragmentTemplate;
+use Contao\StringUtil;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+#[AsContentElement(category: 'texts')]
+class DescriptionListController extends AbstractContentElementController
+{
+    protected function getResponse(FragmentTemplate $template, ContentModel $model, Request $request): Response
+    {
+        $descriptions = [];
+        $data = StringUtil::deserialize($model->data);
+
+        if (!empty($data) && \is_array($data)) {
+            foreach ($data as $row) {
+                $descriptions[] = [
+                    'term' => $row['key'],
+                    'details' => $row['value'],
+                ];
+            }
+        }
+
+        $template->set('descriptions', $descriptions);
+
+        return $template->getResponse();
```

Comment:
```suggestion
        $template->set('descriptions', array_map(
            static function ($row): array {
                return [
                    'term' => $row['key'],
                    'details' => $row['value'],
                ];
            },
            StringUtil::deserialize($model->data, true)
        ));

        return $template->getResponse();
```

Reply:
You can shorten this to an arrow function :wink:.

Reply:
I always rely on the CS fixer or Rector to do it for me and this time it did not 😁